interp/interp_test.go

```
@@ -1299,6 +1546,253 @@ func TestInterpreter_Run(t *testing.T) {
+			instr.New(instr.I32_CONST, 1),
+			instr.New(instr.I32_CONST, 2),
+			instr.New(instr.I32_ADD),
+			instr.New(instr.DROP),
+		)
+	}
+	benchmarkNumeric = append(benchmarkNumeric, instr.New(instr.I32_CONST, 42))
+
+	if runtime.GOARCH == "arm64" {
+		t.Run("I32Add/Straight/JITCold", func(t *testing.T) {
+			vm := New(program.New(benchmarkNumeric), WithTick(1), WithThreshold(0))
+			defer vm.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `vm.Close` is not checked

(errcheck)